README.md

```diff
@@ -62,6 +62,28 @@ All the actions shown between the clusters are done by the konnector, except: th
 
 To get familiar with setting up the environment, please check out docs at [kube-bind.io](https://docs.kube-bind.io/main/setup).
 
+## API Changes in v0.6.0 release
+
+### Catalog API
+Introduction of new `Collection` and `APIServiceExportTemplate` CRDs for better service organization:
+- **Collections**: Function as folders in the UI, grouping related modules
```

Comment:
```suggestion
- **Collections**: Function as folders in the UI, grouping bindable resources such es templates
```

I'd call it abstract resources. Maybe we'll have other bindable resources in the future.